articles/stream-analytics/stream-analytics-real-time-fraud-detection.md

```diff
@@ -169,6 +169,26 @@ The next step is to define an input source for the job to read data using the ev
 
    ![Configure Azure Stream Analytics input](media/stream-analytics-real-time-fraud-detection/configure-stream-analytics-input.png)
 
+## Create a consumer group
+
+It is recommended to use a distinct consumer group for each Stream Analytics job. If no consumer group is specified, the Stream Analytics job uses the $Default consumer group. When a job contains a self-join or has multiple inputs, some inputs might be read by more than one reader downstream. This situation impacts the number of readers in a single consumer group.
```

Comment:
@anboisve can you clarify?
The statement is ambiguous. Are we recommending that users use a distinct consumer group to avoid issues when a job contains self-join or has multiple inputs?  is there any other reason where using a distinct consumer group is ideal? What issues specifically may surface if users fail to use a distinct consumer group and the job contains a self-join or has multiple input?

Can you also clarify this statement: " This situation impacts the number of readers in a single consumer group"
Are you saying that the impact is that some inputs might be read by more than one reader downstream if a job contains a self-join or has multiple inputs? How does this affect the job/results if that is the case?

Reply:
#assign: @anboisve

Reply:
The recommendation is to create a distinct consumer group for each job, this will minimize issues with more complex queries (i.e. sql self-join), or if the customer ends up having multiple inputs for a single job, then they should definitely add a consumer group.  I ran into this issue with my job when my query got more complex, adding the consumer group to my event hub solved the issue.

Reply:
What issue did you run into? I am trying to make it clearer to customers what the actual implications will be if they don't use a distinct consumer group. is it duplication, low throughput, or anything in particular?